app/managers.py

```diff
@@ -0,0 +1,45 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self.db_connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, second_name: str) -> None:
+        self.db_connection.execute(
+            f"INSERT INTO {self.table_name} (first_name, second_name)"
+            f" VALUES (?, ?)",
+            (first_name, second_name)
+        )
+
+        self.db_connection.commit()
+
+    def all(self) -> list:
+        list_of_actors = self.db_connection.execute(f"SELECT * FROM"
+                                                    f" {self.table_name}")
+        return [Actor(*i) for i in list_of_actors]
```

Comment:
The `all` method should fetch all rows from the database, but the current implementation might not work as expected. The `execute` method returns a cursor, which needs to be iterated or fetched to get the actual data. Consider using `fetchall()` to retrieve the data.